src/Kmd.Logic.CitizenDocuments.Client/CitizenDocumentsOptions.cs

```diff
@@ -15,10 +13,8 @@ public sealed class CitizenDocumentsOptions
         /// <remarks>
         /// This option should not be overridden except for testing purposes.
         /// </remarks>
-        public Uri Serviceuri { get; set; } = new Uri("https://gateway.kmdlogic.io/cpr/v1");
-
-        public string SubscriptionId { get; set; } = "";
-
+        public Uri Serviceuri { get; set; } = new Uri(string.Empty);
 
+        public string SubscriptionId { get; set; } = string.Empty;
```

Comment:
What makes this better than `null` ?

Reply:
It does not but we are putting for now as a placeholder and they can be replaced by developers as an when required